src/client/src/core/util/validation/rules.ts

```diff
@@ -73,6 +75,18 @@ export const validDateFormat = function(dateFormat = 'yyyy-MM-dd') {
   };
 };
 
+export const shouldBeCurrentOrPreviousDate = function() {
```

Comment:
Need to add unit tests for this

Reply:
Please check https://github.com/orangehrm/orangehrm/blob/main/src/client/src/core/util/validation/__tests__/rules.spec.ts